encoding/kzg/verifier/batch_commit_equivalence_test.go

```diff
@@ -14,17 +14,21 @@ import (
 )
 
 func TestBatchEquivalence(t *testing.T) {
```

Comment:
It'll be nice to have a icicle-gnark equivalence test for proofs, but may need gpu setup for unit test... Maybe we can start with CPU (for both), in a follow up